src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -638,6 +638,12 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                 profiles.map(profile=>profile.molecularAlterationType)
             ).filter(type=>!!dataTypeToDisplayType[type]); // only show profiles of the type we want to show
 
+            // if no gene sets are queried, remove gene set profile from dataTypeIds
+            if (this.props.store.genesets.result!.length == 0) {
+                    let index = dataTypeIds.indexOf("GENESET_SCORE");
```

Comment:
dataTypeIds.includes("GENESET_SCORE")

Reply:
`includes` returns a boolean, how do I delete then the element from the array?